.github/workflows/build_fork.yml

```diff
@@ -207,8 +207,8 @@ jobs:
           # run this in CI if it gets an incorrect lake hash for existing cache files somehow
           # lake exe cache pack! || true
           lake exe cache commit || true
-          # try twice in case of network errors
-          lake exe cache put || (sleep 1; lake exe cache put) || true
+          # do not try to upload files just downloaded; try twice in case of network errors
+          lake exe cache put-unpacked || (sleep 1; lake exe cache put-unpacked) || true
```

Comment:
will this still work if using `cache pack!` above?

Reply:
Yeah I really wanted to avoid additional bookkeeping... does the "somehow" case described still happen or was it from previous bugs? And shouldn't we just run `put!` in that rare case?

I rebase onto master which removed the second invocation

Reply:
The "somehow" case can still happen, in the sense that if due to a weird bug an incorrect cache file is uploaded, the system will not self-correct and that `pack!` is supposed to replace all cache files with known-good files. I don't mind changing the mechanism to do this though.